packages/echo-base/src/module/dependency.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { AvailableDependencies, ModuleData, ModuleMetaData } from '../types';
+
+declare global {
+    interface HTMLScriptElement {
+        module?: ModuleData;
+    }
+}
+
+/**
+ * Returns the module dependency if present in the AvailableDependencies
+ *
+ * @param {string} name
+ * @param {AvailableDependencies} dependencies
+ * @return {*}  {*}
+ */
+function requireModule(name: string, dependencies: AvailableDependencies): any {
```

Comment:
Funksjonen heter noe helt annet enn hva som står som comment.
rename til getDependency? Skal retur type være any?

Reply:
kan sikkert være unknown etter som denne funksjonen legges på window objektet å bukes av module for å laste Dependency